doc/en/components/grids/_shared/export-excel.md

```diff
@@ -0,0 +1,234 @@
+
+## Export All Data
+
+There are some cases when you might be using remote operations like *Paging* and the Grid won't have access to all of its data. In these cases, we recommend using the [Excel Export Service](../exporter-excel.md) and pass the whole data collection, if available. Example:
+
+```ts
+public exportButtonHandler() {
+  this.excelExportService.exportData(this.localData, new ExcelExporterOptions('ExportedDataFile'));
+}
+```
+
+<!-- ComponentStart: Grid -->
+## Export Grouped Data
+
+To export grouped data you just need to group the {Component} by one or more columns. The browser will download a file named "ExportedDataFile.xlsx" which contains the data from the {Component} component in MS Excel format grouped by the selected column. Example:
+
+
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz" 
+           iframe-src="{environment:demosBaseUrl}/services/export-excel-sample-1" alt="{Platform} Grouped Data Excel Exporter Example">
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+## Export Multi Column Headers Grid
+
+It is now possible to export {Component} with defined [multi-column headers](multi-column-headers.md). All headers will be reflected in the exported excel file as they are displayed in the {Component}. If you want to exclude the defined multi-column headers from the exported data you can set the [exporter option]({environment:demosBaseUrl}/classes/exporteroptionsbase.html) [ignoreMultiColumnHeaders]({environment:demosBaseUrl}/classes/exporteroptionsbase.html#ignoremulticolumnheaders) to `true`.
+
+> [!NOTE]
+> The exported {Component} will not be formatted as a table, since Excel tables do not support multiple row headers.
+
+<!-- ComponentStart: Grid -->
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz"
+           iframe-src="{environment:demosBaseUrl}/grid/multi-column-headers-export" alt="{Platform} Multi-Column Headers Export">
+</code-view>
+<!-- ComponentEnd: Grid -->
+
+<!-- ComponentStart: TreeGrid -->
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz" 
+           iframe-src="{environment:demosBaseUrl}/tree-grid/treegrid-multi-column-headers-export" alt="{Platform} Multi-Column Headers Export">
+</code-view>
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+<code-view style="height: 800px;" 
+           data-demos-base-url="{environment:demosBaseUrl}"
+           explicit-editor="stackblitz"
+           iframe-src="{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-multi-column-export" alt="{Platform} Multi-Column Headers Export">
+</code-view>
+<!-- ComponentEnd: HierarchicalGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.